DefaultCoreForEventBusTest#concurrentUseAbilityTest may fail

- [ ] I have searched the [issues](https://github.com/seata/seata/issues) of this repository and believe that this is not a duplicate.

### Ⅰ. Issue Description


### Ⅱ. Describe what happened

  If there is an exception, please attach the exception trace:

```
io.seata.server.event.DefaultCoreForEventBusTest
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   LockManagerTest.concurrentUseAbilityTest:145 expected: <true> but was: <false>
[INFO] 
[ERROR] Tests run: 90, Failures: 1, Errors: 0, Skipped: 1
```


### Ⅲ. Describe what you expected to happen


### Ⅳ. How to reproduce it (as minimally and precisely as possible)


### Ⅴ. Anything else we need to know?


### Ⅵ. Environment:

- JDK version :
- OS :
- Others:

--- a/server/src/test/java/io/seata/server/lock/LockManagerTest.java
+++ b/server/src/test/java/io/seata/server/lock/LockManagerTest.java
@@ -142,7 +142,7 @@ public class LockManagerTest {
             }).start();
             // Assume execute more than 5 seconds means deadlock happened.
             if (countDownLatch.await(5, TimeUnit.SECONDS)) {
-                Assertions.assertTrue((first.get() && !second.get()) || (second.get() && !first.get()));
+                Assertions.assertTrue(!first.get() || !second.get());
             }
         } finally {
             lockManager.releaseLock(branchSession1);

